src/linux_mcp_server/audit.py

```diff
@@ -90,80 +96,109 @@ def process(self, msg, kwargs):
     yield adapter
 
 
-def log_tool_call(tool_name: str, parameters: dict[str, t.Any]):
-    """
-    Log a tool invocation.
-
-    Args:
-        tool_name: Name of the tool being called
-        parameters: Tool parameters (will be sanitized)
-    """
-    logger = logging.getLogger(__name__)
-
-    # Determine if local or remote execution
-    execution_mode = "remote" if parameters.get("host") else "local"
-
-    # Sanitize parameters
-    safe_params = sanitize_parameters(parameters)
+def _log_event_start(
+    logger: logging.Logger,
+    tool_name: str,
+    execution_mode: str,
+    params: dict[t.Any, t.Any],
+) -> int:
```

Comment:
Does this need a docstring? Especially considering it's returning a timespan of some sort. We should definitely document what the timespan means.